Server.java

```diff
@@ -0,0 +1,82 @@
+import static my.Constants.*;
+import static my.HardCodes.*;
+
+import org.json.*;
+import java.util.*;
+import java.io.*;
+import javax.net.ssl.SSLContext;
+import javax.net.ssl.SSLServerSocketFactory;
+import javax.net.ssl.SSLServerSocket;
+import javax.net.ssl.SSLSocket;
+
+public class Server {
+  public static void main(String[] args) throws JSONException, IOException {
+    ArrayList<ErrorInfo> pids = new ArrayList<ErrorInfo>();
+  
+    try {
+      InitSSL initSsl = new InitSSL();
```

Comment:
Ideally you would not be working at this low of a level and instead use something like HTTP/JSON or GRPC where a lot of this work would be abstracted away from you.

However, since it appears you are implementing a protocol (JSON-RPC), you should abstract this out yourself. 

Most of this code should sit in it's own class with a simple interface that you can use to perform a request and get a response. That interface is what you should use here.

Reply:
I'm not sure if this still applies with the changes that I've made. Let me know.